Add notes and handlers attribute in agent /events API example (#2657)
sensu/sensu-docs: content/sensu-go/5.21/reference/agent.md

The request creates event for a check named `check-mysql-status` with the output `could not connect to mysql` and a status of `1` (warning).
The agent responds with an HTTP `202 Accepted` response to indicate that the event has been added to the queue to be sent to the backend.

The event will be handled according to an `email` handler definition.

{{% notice note %}}
**NOTE**: For HTTP `POST` requests to the agent /events API, check [spec attributes](../checks/#spec-attributes) are not required.

> In that case, the spec attributes (including `handlers`) are listed as individual [top-level attributes](../checks/#top-level-attributes) instead, and there is no `spec` object in the check definition.

Author: ```suggestion
When doing so, the spec attributes (including `handlers`) are listed as individual [top-level attributes](../checks/#top-level-attributes) instead, and there is no `spec` object in the check definition.
```

> When doing so, the spec attributes (including `handlers`) are listed as individual [top-level attributes](../checks/#top-level-attributes) instead, and there is no `spec` object in the check definition.

Author: ```suggestion
When doing so, the spec attributes (including `handlers`) are listed as individual [top-level attributes](../checks/#top-level-attributes) in the check definition instead.
```
